PWGDQ/Core/VarManager.h

```diff
@@ -1024,8 +1034,8 @@ KFPVertex VarManager::createKFPVertexFromCollision(const T& collision)
   kfpVertex.SetXYZ(collision.posX(), collision.posY(), collision.posZ());
   kfpVertex.SetCovarianceMatrix(collision.covXX(), collision.covXY(), collision.covYY(), collision.covXZ(), collision.covYZ(), collision.covZZ());
   kfpVertex.SetChi2(collision.chi2());
-  kfpVertex.SetNDF(2 * collision.multNTracksPV() - 3);
-  kfpVertex.SetNContributors(collision.multNTracksPV());
+  kfpVertex.SetNDF(2 * collision.numContrib() - 3);
+  kfpVertex.SetNContributors(collision.numContrib());
```

Comment:
why is this change needed ?

Reply:
Hi Ionut,
This is a bug fix when creating the KF object for the PV. In principle, the variable need to be used to set the NDF and NContributors to the PV is how many tracks were used to reconstruct the PV. By mistake, the `multNTracksPV` which means removing the number of the secondary tracks from the `numContrib` was used.